api/team.go

```diff
@@ -801,6 +830,10 @@ func updateTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	oldTeam.Sanitize()
 
+	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_UPDATE_TEAM, "", "", "", nil)
```

Comment:
Why a message for Inviting?  the user hasn't joined yet??? or am I missing something.